app/assets/stylesheets/common.scss.erb

```diff
@@ -108,9 +108,16 @@
   }
 }
 
-.#{$namespace}-stanford-only a{
-  @include stanford-only;
-  padding-right: 20px;
+.#{$namespace}-stanford-only a {
```

Comment:
Yes.  This is basically saying, style _an anchor tag within an element classed `.sul-embed-stanford-only`_.